sql/moz-fx-data-shared-prod/mozilla_vpn/vat_rates/view.sql

```diff
@@ -2,6 +2,8 @@ CREATE OR REPLACE VIEW
   `moz-fx-data-shared-prod.mozilla_vpn.vat_rates`
 AS
 SELECT
-  *
+  *,
+  LAG(effective_date) OVER (PARTITION BY country ORDER BY effective_date) AS prev_effective_date,
+  LEAD(effective_date) OVER (PARTITION BY country ORDER BY effective_date) AS next_effective_date,
```

Comment:
these functions do the exact opposite of what I expect, but I'm fairly certain I'm using the correct ones: https://cloud.google.com/bigquery/docs/reference/standard-sql/navigation_functions#lead